### ISSUE TITLE
[FEATURE REQUEST] - Item exclusion from playlist download and tweaks

### ISSUE THREAD

**Issue by J316** [rg3/youtube-dl#11508]
If I want to download all the playlists of a given channel but one, I have to put all their links in the command, it would be way more convenient to just put the playlist section of that channel and then add the link of the one I want to ignore with an exclusion command. I think downloading playlists should be more _"tweakable"_ since various scenarios can present.

**Comment by shizeeg** [comment]
you may try to use:
`$ youtube-dl https://yoursite.com/bla-bla --playlist-items 1-9,11,16`

like that.

**Comment by yan12125** [comment]
Thanks @shizeeg!

**Comment by J316** [comment]
@yan12125
That's totally wrong, I was not talking about playlist items, but about playlists of a given channel.

**Comment by yan12125** [comment]
Could you give some examples of channels and playlists?

**Comment by J316** [comment]
Open any channel with more than a playlist in the playlists section and there you go.

**Comment by yan12125** [comment]
Which website? YouTube? If so could you please provide an example link? That may sound ironic, but I'm not familiar with YouTube.

**Comment by J316** [comment]
[And there you go](https://www.youtube.com/channel/UCLW0TEV3YEt-J56pYZm-TgA)

**Comment by yan12125** [comment]
Did you mean you want to choose specific playlists on https://www.youtube.com/channel/UCLW0TEV3YEt-J56pYZm-TgA/playlists instead of downloading all 10 playlists?

**Comment by J316** [comment]
Yes. It's way inconvenient with channels with many playlists.

**Comment by yan12125** [comment]
Thanks I got you. 

Related to #10122

**Comment by J316** [comment]
It seems to me it's not related at all to #10122

**Comment by yan12125** [comment]
Currently
```
youtube-dl -vF https://www.youtube.com/channel/UCLW0TEV3YEt-J56pYZm-TgA/playlists
```
downloads all videos of all 10 playlists. And
```
youtube-dl -vF https://www.youtube.com/channel/UCLW0TEV3YEt-J56pYZm-TgA/playlists --playlist-items 1
```
download the first video of the first playlist. It should download all videos of the first playlist, just like the case in #10122
